doc/subsystems/networking/usbnet_setup.rst

```diff
@@ -0,0 +1,79 @@
+.. _usb_device_networking_setup:
+
+USB Device Networking
+#####################
+
+This page describes how to set up networking between a Linux host
+and a Zephyr application running on USB supported devices. A sample
```

Comment:
Is this just a network connection between the board and host?  Is this a way to share the host's IP address and give the board internet access?  Maybe explain more about what this network connection is for and what you can do with it.

Reply:
At the moment this is a network connection between the board and host.